pacai/bin/capture.py

```diff
@@ -511,75 +510,90 @@ def parseAgentArgs(str):
-            help=default('Save the game as a gif to the specified path'))
-    parser.add_option('--gif-skip-frames', dest='gifSkipFrames', type='int', default=0,
-            help=default('Skip this number of frames between frames of the gif.'))
-    parser.add_option('--gif-fps', dest='gifFPS', type='float', default=10,
-            help=default('FPS of the gif.'))
-
-    options, otherjunk = parser.parse_args(argv)
-    assert len(otherjunk) == 0, "Unrecognized options: " + str(otherjunk)
+
+    parser = getParser(description, os.path.basename(__file__))
+
+    parser.add_argument('-b', '--blue', dest = 'blue',
+            action = 'store', type = str, default = 'pacai.core.baselineTeam',
+            help = 'set blue team (default: %(default)s)')
+
+    parser.add_argument('-l', '--layout', dest = 'layout',
+            action = 'store', type = str, default = 'defaultCapture',
+            help = 'use the specified map layout or input RANDOM<seed> '
+                + 'for a random seeded map (i.e. RANDOM23) (default: %(default)s)')
+
+    parser.add_argument('-r', '--red', dest = 'red',
+            action = 'store', type = str, default = 'pacai.core.baselineTeam',
+            help = 'set red team (default: %(default)s)')
+
+    parser.add_argument('--blue-args', dest = 'blueArgs',
+            action = 'store', type = str, default = None,
+            help = 'comma separated arguments to be passed to blue team (e.g. \'opt1=val1,opt2\') '
+                + '(default: %(default)s)')
+
+    parser.add_argument('--keys0', dest = 'keys0',
+            action = 'store_true', default = False,
+            help = 'make agent 0 (first red player) a keyboard agent (default: %(default)s)')
+
+    parser.add_argument('--keys1', dest = 'keys1',
+            action = 'store_true', default = False,
+            help = 'make agent 1 (first blue player) a keyboard agent (default: %(default)s)')
+
+    parser.add_argument('--keys2', dest = 'keys2',
+            action = 'store_true', default = False,
+            help = 'make agent 2 (second red player) a keyboard agent (default: %(default)s)')
+
+    parser.add_argument('--keys3', dest = 'keys3',
+            action = 'store_true', default = False,
+            help = 'make agent 3 (second blue player) a keyboard agent (default: %(default)s)')
+
+    parser.add_argument('--max-moves', dest = 'maxMoves',
+            action = 'store', type = int, default = 1200,
+            help = 'set maximum number of moves in a game (default: %(default)s)')
+
+    parser.add_argument('--red-args', dest = 'redArgs',
+            action = 'store', type = str, default = None,
+            help = 'comma separated arguments to be passed to red team (e.g. \'opt1=val1,opt2\') '
+                + '(default: %(default)s)')
+
+    options, otherjunk = parser.parse_known_args(argv)
     args = dict()
 
+    if len(otherjunk) != 0:
+        raise ValueError('Unrecognized options: \'%s\'. ' % (str(otherjunk)))
```

Comment:
There is an extra space after the period.
Also, why not use double quotes for the inner quotes?